oneapp-server/types/ApplicationContact.js

```diff
@@ -0,0 +1,91 @@
+const { gql } = require('apollo-server-express');
+
+const typeDef = gql`
+  extend type Application {
+    "The individual applying for benefits"
+    contact: ApplicationContact
+  }
+
+  type ApplicationContact {
+    "Applicant first name"
+    APPLICANT_FIRST_NAME: String
+    "Applicant last name"
+    APPLICANT_LAST_NAME: String
+    "Applicant middle name"
+    APPLICANT_MIDDLE_NAME: String
+    "Application maiden name"
+    APPLICANT_MAIDEN_NAME: String
+    "Payee first name, if different than applicant"
+    PAYEE_FIRST_NAME: String
+    "Payee last name, if different than applicant"
+    PAYEE_LAST_NAME: String
```

Comment:
Fields like this will be interesting, as they're optional but required under certain circumstances. In this case, there is no db field to indicate the applicant is not the payee, that's done completely in UI in the current version of OneApp, but if they're not, then this information must be filled out.